translations/en-us.yml

```diff
@@ -2187,12 +2187,12 @@ osf-components:
         acceptRevisionDescription: 'Update will be accepted and registration will be updated'
         rejectRevision: 'Reject update'
         rejectRevisionDescription: 'Update will be rejected and registration will remain unchanged'
-        acceptCollectionSubmission: 'Accept submission'
-        acceptCollectionSubmissionDescription: 'Project will appear in search results and be associated with the collection'
-        rejectCollectionSubmission: 'Reject submission'
-        rejectCollectionSubmissionDescription: 'Project will not appear in search results nor be associated with the collection'
-        removeCollectionSubmission: 'Remove submission'
-        removeCollectionSubmissionDescription: 'Collection will be removed from the collection and no longer appear in search results'
+        acceptCollectionSubmission: 'Accept Request'
+        acceptCollectionSubmissionDescription: 'Request will appear in search results and be associated with the collection'
```

Comment:
I think for this one, `Project will appear in search results...` is ok, unless Product folks wanted this one updated too.